src/kixi/datastore/metadatastore/command/handler.clj

```diff
@@ -69,15 +74,17 @@
   [payload]
   {::ke/key ::kdm/update-rejected
    ::ke/version "1.0.0"
-   ::ke/payload payload})
+   ::ke/payload payload
+   ::ke/partition-key (or (::ms/id payload)
+                          (get-in payload [:original ::ms/id]))})
 
 (defn invalid
   ([cmd speccy data]
    (invalid cmd (spec/explain-data speccy data)))
   ([cmd explanation]
    (update-rejected {:reason :invalid
                      :explanation explanation
-                     :original {::ms/payload cmd}
+                     :original (::ms/payload cmd)
```

Comment:
we *could* get rid, but needs to be coordinated with frontend - I'd put it back for now